packages/admin-next/dashboard/src/routes/products/product-create/components/product-create-variants-form/product-create-variants-form.tsx

```diff
@@ -19,6 +20,8 @@ type ProductCreateVariantsFormProps = {
 export const ProductCreateVariantsForm = ({
   form,
 }: ProductCreateVariantsFormProps) => {
+  const { regions } = useRegions()
```

Comment:
This would causes issues if the store has more regions then what we return by default. Properly very unlikely but could happen if people have created a region for all countries they sell to. 

I think we should set the limit to something like 999, and only select the fields that we require for the editor to work, so id, title, and currency_code.

Reply:
I intended to do that but I passed 9999 limit to the store hook below 🤦